diff --git a/src/Makefile b/src/Makefile
index 9f36cc5562..7a5a8ffa45 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -794,15 +794,9 @@ endif
 
 ${WAZUH_LIB_OUTPUT_PATH}${LIBSTDCPP_NAME}: ${libstdc++_path}
 	cp $< $@
-ifneq (${uname_S},AIX)
-	${STRIP_TOOL} -x $@
-endif
 
 ${WAZUH_LIB_OUTPUT_PATH}${LIBGCC_S_NAME}: ${libgcc_s_path}
 	cp $< $@
-ifneq (${uname_S},AIX)
-	${STRIP_TOOL} -x $@
-endif
 
 .PHONY: server local hybrid agent selinux
 
@@ -955,7 +949,7 @@ endif
 ifeq ($(CHECK_CENTOS5),YES)
 EXTERNAL_LIBS += $(PROCPS_LIB)
 else
-EXTERNAL_LIBS += $(PROCPS_LIB) $(LIBALPM_LIB) $(LIBARCHIVE_LIB)
+EXTERNAL_LIBS += $(PROCPS_LIB) $(LIBALPM_LIB) $(LIBARCHIVE_LIB) $(DB_LIB)
 endif
 endif
 endif
@@ -1025,7 +1019,7 @@ else
 	cd ${EXTERNAL_OPENSSL} && ./Configure $(OPENSSL_FLAGS) solaris-sparcv9-gcc && ${MAKE} build_libs
 endif
 else
-	cd ${EXTERNAL_OPENSSL} && ./config $(OPENSSL_FLAGS) && ${MAKE} build_libs
+	cd ${EXTERNAL_OPENSSL} && perl ./Configure $(OPENSSL_FLAGS) && ${MAKE} build_libs
 endif
 endif
 endif
